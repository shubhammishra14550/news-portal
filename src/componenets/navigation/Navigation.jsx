import { Navbar, Container, Nav } from "react-bootstrap";
import React from "react";

export default function Navigation() {
  return (
    <Navbar bg="dark" variant="dark">
      <Container>
        <Navbar.Brand href="#bulletin">News Express</Navbar.Brand>
        <Nav className="me-auto">
          <Nav.Link href="#bulletin">Bulletin</Nav.Link>
          <Nav.Link href="#addNews">Add News</Nav.Link>
          <Nav.Link href="#filterNews">Filter News</Nav.Link>
        </Nav>
      </Container>
    </Navbar>
  );
}
