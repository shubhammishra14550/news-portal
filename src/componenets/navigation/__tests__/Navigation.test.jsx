import { render } from '@testing-library/react';
import Navigation from '../Navigation';

describe('Navigation component', () => {
    it('should renders correctly', () => {
        const { asFragment } = render(<Navigation />);
        expect(asFragment()).toMatchSnapshot();
    })
})
