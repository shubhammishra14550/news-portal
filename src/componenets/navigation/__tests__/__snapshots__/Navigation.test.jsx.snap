// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component should renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand navbar-dark bg-dark"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="#bulletin"
      >
        News Express
      </a>
      <div
        class="me-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rr-ui-event-key="#bulletin"
          href="#bulletin"
        >
          Bulletin
        </a>
        <a
          class="nav-link"
          data-rr-ui-event-key="#addNews"
          href="#addNews"
        >
          Add News
        </a>
        <a
          class="nav-link"
          data-rr-ui-event-key="#filterNews"
          href="#filterNews"
        >
          Filter News
        </a>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
